common/reviews/api/operation-graph.api.md

```diff
@@ -99,7 +99,7 @@ export interface IRequestRunEventMessage {
     // (undocumented)
     event: 'requestRun';
     // (undocumented)
-    requestor?: string;
+    requester: string;
```

Comment:
"requestor" is a word: https://grammarhow.com/requester-or-requestor/

Reply:
followed @octogonz linguistic guidance from the original PR, but I'm happy to reduce the change impact.